.github/workflows/release.yml

```diff
@@ -55,64 +64,61 @@ jobs:
           gpg-private-key: ${{ secrets.MAVEN_GPG_KEY }}
           gpg-passphrase: MAVEN_GPG_PASSPHRASE
 
-      - name: Publish SNAPSHOT
-        run: mvn -B --no-transfer-progress clean deploy
-        env:
-          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
-          MAVEN_PASSWORD: ${{ secrets.OSSRH_PASSWORD }}
-          MAVEN_GPG_PASSPHRASE: ${{ secrets.MAVEN_GPG_PASSPHRASE }}
-
-  # Run for manual trigger (workflow dispatch), since you'll have release and next dev versions specified
-  # All commits will have a -SNAPSHOT project version anyway, since the releases will be handled here
-  release:
-    needs: setup
-    if: github.event_name == 'workflow_dispatch'
-    runs-on: ubuntu-latest
-    env:
-      RELEASE: ${{ inputs.releaseversion }}
-      NEXT: ${{ inputs.nextversion }}
-    steps:
-      - name: Checkout latest code
-        uses: actions/checkout@v3
-
       - name: Config git user
         run: |
           git config user.name ${{ github.actor }}
           git config user.email "${{ github.actor }}@users.noreply.github.com"
 
-      - name: Setup Java & Maven
-        uses: actions/setup-java@v3
-        with:
-          java-version: 17
-          distribution: 'temurin'
-          cache: 'maven'
-          server-id: ossrh
-          server-username: MAVEN_USERNAME
-          server-password: MAVEN_PASSWORD
-          gpg-private-key: ${{ secrets.MAVEN_GPG_KEY }}
-          gpg-passphrase: MAVEN_GPG_PASSPHRASE
-      # =============================================================================
-      # Start the release
-      # =============================================================================
-      - name: Release main POM
+      - name: Set the release version, commit the change, and tag it
         run: |
-          mvn -B -U -V -ntp release:prepare -DreleaseVersion=$RELEASE -Dtag=$RELEASE -DdevelopmentVersion=$NEXT -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
-          mvn -B -U -V -ntp release:perform -P release -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+          mvn versions:update-parent -B -ntp -DparentVersion=$RELEASE
+          mvn versions:set -B -ntp -DnewVersion=$RELEASE
+          git commit -am "Update version to $RELEASE"
+          git tag $RELEASE
+
+      - name: Release Java modules
+        uses: eclipse-pass/main/.github/actions/maven-release@combined-release
```

Comment:
This will need to be updated to point to `main`, not sure if we want to do that now or after the PR is merged

Reply:
Shoot. I thought I had caught all of those. Will update. Yes. Various workflows will fail until they get merged.